### ISSUE TITLE
Can we have a frog discussion mailing list?

### ISSUE THREAD

**Issue by pmatos** [greghendershott/frog#187]
I actually thought there was one and I was so convinced this was true that I kept digging but found no references to it. I assume there is non, @greghendershott would you be able to create one?

**Comment by greghendershott** [comment]
If Frog were to need its own mailing list then I'd need a lot more contributors. 😛 

Seriously:

- I haven't had much time to keep up with Frog in the last couple years. The last few weeks, I've had an opportunity to do more, but I don't know how long this period will last.
- Generally speaking if someone pings me on IRC or the Racket mailing list about Frog (or racket-mode), I'll ask them to file an issue here. Often I'll do that for them. For me it's a much better "inbox" and tracking tool.
- I don't mind if some issues here turn into long, wandering discussion threads. I can put a `question` label on them, and later split out any specific to-do items as separate issues.

**Comment by pmatos** [comment]
Fair enough. I understand. Lets close this then. I think then allowing Issues to be questions, might be a good alternative.

**Comment by krrrcks** [comment]
Yes, sounds like a good plan. Maybe we could use the issues (or the wiki) for "how I did this and that with Frog" as well? 

That would be interesting at least for me.

**Comment by tger** [comment]
To me this sounds overly contrived. There is apparently a need for Frog users to communicate, and apart from the fact that it's different from more or less every other project out there expecting everyone to "watch" the source repository does not seem intuitive to me. I created an issue (#179) to try to reach out to the Frog community and while that feature may be not be that attractive to most Frog users I do find it a bit odd that no-one has made any comment whatsoever.

Creating a mailing list or a discussion forum is very easy and in the non-moderated case (which should be fine I think) I cannot see any downsides. If @greghendershott is afraid that he will not be able to keep on top of things there (which I don't think is a necessary) I am volunteering to take on this role.

**Comment by tfeb** [comment]
What proportion of Frog users who want to discuss it are not watching the GH repo?

In my case: I am, but I would be extremely unlikely to subscribe to yet another mailing list because what I care about is changes to Frog which I might need to take account of, and I get to know abotu those by watching the repo.  Others may differ I suppose.

**Comment by pmatos** [comment]
It really sounded to me like @greghendershott initial suggestion of using issues for questions might work quite well with the small community we have.

The wiki is well suitable for articles like `how to port your frog to bootstrap 4` which I will try to write by this weekend as I should be soon uploading my new website based on frog and bootstrap4.

@tger As for people not commenting on #179 , well I have to say I read all open issues with frog and commented in almost none. As for yours, I simply have no idea what responsive images are and since images are the least of my concerns I didn't comment or show interest. My apologies for that. :) That doesn't obviously mean I won't need responsive images in the future but at the moment the least amount of images on my site, the better.

**Comment by krrrcks** [comment]
Looks like a good idea. I think we can extend the Wiki with Howtos and code snippets (I am going to provide one later).

**Comment by greghendershott** [comment]
@krrrcks Thanks for [this](https://github.com/greghendershott/frog/wiki/Snippets#show-tags-feeds-in-navigation-bar-depending-on-url).

**Comment by tger** [comment]
@pmatos See for instance https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images

**Comment by lojic** [comment]
FYI - There is now a frog channel in the Racket Slack.
